scripts/kong-db-checker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+if pg_isready -U postgres -h kong-db; then
```

Comment:
Shouldn't this be
```
pg_isready -h kong-db -d kong -U kong
```

My edinburgh instance of kong-db does not have a postgres user in it, only kong.

Reply:
Fair enough

Reply:
Done